server/data/educationAndWorkPlanClient.test.ts

```diff
@@ -323,17 +324,15 @@ describe('educationAndWorkPlanClient', () => {
       const systemToken = 'a-system-token'
       const updateInductionRequest = aShortQuestionSetUpdateInductionRequest()
 
-      const expectedResponseBody = {}
-      educationAndWorkPlanApi
+      educationAndWorkPlanApi //
         .put(`/inductions/${prisonNumber}`, updateInductionRequest)
-        .reply(204, expectedResponseBody)
```

Comment:
This test (for the Update method in the client class) was wrong because the API for Update (and Create) do not return a response body, so it was wrong/misleading to setup the mock to do so